hydra-node/test/Hydra/CryptoSpec.hs

```diff
@@ -68,11 +70,18 @@ specMultiSignature =
     prop "is sensitive to order" $ \(allSigs :: HashSet (Signature ByteString)) ->
       let sigs = toList allSigs
        in forAll (shuffle sigs) $ \shuffled ->
-            (length sigs > 1 && sigs /= shuffled) ==>
-              (aggregate sigs =/= aggregate shuffled)
+            (length sigs > 1 && sigs /= shuffled)
+              ==> (aggregate sigs =/= aggregate shuffled)
 
     prop "aggregate/verifyMultiSignature roundtrip" $ \sks (msg :: ByteString) ->
-      let sigs = map (\sk -> sign sk msg) sks
+      let sigs = map (`sign` msg) sks
           msig = aggregate sigs
           vks = getVerificationKey <$> sks
-       in verifyMultiSignature vks msig msg
+       in verifyMultiSignature vks msig msg === Verified
+
+    prop "aggregateInOrder/verifyMultiSignature roundtrip" $ \sks (msg :: ByteString) ->
```

Comment:
We probably want to test also what happens when signatures are not verified.

Reply:
Good catch. Something to do in another PR?